Automattic/jetpack
# VideoPress: add dialog to convert core/video to videopress/video

This PR adds a dialog to the `core/video` extended block, AKA VideoPress block, that suggests the user switch to the new VideoPress video block `v6`.
This is a transition case since we'd like to either replace the current implementation with `v6`

Part of https://github.com/Automattic/jetpack/issues/26021

#### Changes proposed in this Pull Request:

* VideoPress: add dialog to convert core/video to videopress/video

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Test with Jetpack plugin active. Activate VideoPress standalone is optional
* Create a `core/video` instance, uploading a view
* Confirm you see the dialog
* Confirm you can keep using the current implementation, or replace it by `v6`

https://user-images.githubusercontent.com/77539/195181452-f8d1ee80-cbff-4949-8f4e-2695eff30911.mov
